config/services.yaml

```
@@ -7,7 +7,8 @@ parameters:
     server_timezone: '%env(APP_SERVER_TIMEZONE)%'
     client_timezone: '%env(APP_CLIENT_TIMEZONE)%'
     admin_email: '%env(ADMIN_EMAIL)%'
-    features: {}
+    features: 
+        road_type_enabled: '%env(bool:FEATURE_ROAD_TYPE_ENABLED)%' 
```

L'idée c'était plutôt de nommer les features par leur nom ici

```suggestion
        road_type: '%env(bool:FEATURE_ROAD_TYPE_ENABLED)%' 
```

Comme ça on fait isFeatureEnabled('road_type') et pas isFeatureEnabled'road_type_enabled') (un peu redondant)